### ISSUE TITLE
app configuration docker-compose

### ISSUE THREAD

**Issue by judos** [nbr23/youtube-dl-server#45]
Hi, I think the repository REDAME is confusing regarding the configuration. I tried the following docker-compose.yml:
```
 youtube-dl:
    image: "nbr23/youtube-dl-server:latest"
    volumes:
      - $HOME/youtube-dl:/youtube-dl
      - ./config.yml:/usr/src/app/config.yml:ro # Overwrite the container's config file with your own configuration
    restart: always
```
This is the one you provided in the README. But then I got the following error:
```
During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/src/app/./youtube-dl-server.py", line 8, in <module>
    from ydl_server.logdb import JobsDB, Job, Actions, JobType
  File "/usr/src/app/ydl_server/logdb.py", line 3, in <module>
    from ydl_server.config import app_config
  File "/usr/src/app/ydl_server/config.py", line 49, in <module>
    app_config = load_config()
  File "/usr/src/app/ydl_server/config.py", line 30, in load_config
    copy_default_config(config_file_path)
  File "/usr/src/app/ydl_server/config.py", line 10, in copy_default_config
    raise Exception("Error copying default config file to {}:\n{}".format(
Exception: Error copying default config file to '/usr/src/app'/config.yml:
[Errno 2] No such file or directory: "'/usr/src/app'/config.yml"
```
Which makes sense because the config.yml is readonly for the container. But also I don't want the container to overwrite the config I pass in. Besides the environment variable `- YDL_CONFIG_PATH='/usr/src/app'` would then also need to be set.
I see it defaults to `ENV YDL_CONFIG_PATH='/app_config'` in the Dockerfile.
So basically I modified my docker-compose to:
```
version: '3'
services:
  ydl:
    container_name: ydl
    image: "nbr23/youtube-dl-server:latest"
    restart: always
    network_mode: bridge
    ports:
      - 8200:8080
    volumes:
      - ./config.yml:/app_config/config.yml:ro #overwrite container config
      - /hd/samba/youtube-dl:/youtube-dl
    environment:
      - TZ=Europe/Zurich
```

And with this config it seems to be working now :)

Can you update the README in the repository?

**Comment by nbr23** [comment]
Thanks a bunch for finding this, fixed and closed :)
